src/App.css

```diff
@@ -0,0 +1,4 @@
+.leaflet-container {
```

Comment:
_Optional_

Since it's in App.css, and not index.css, and I assume that map sizing is specific to a particular instance of map on the webpage, I'd advise this to be scoped within `.App` class itself:

```
.App .leaflet-container {
```

Even better, you could move this declaration to `index.css`, specifying `height:100%` instead of `height:100vh`, and then manipulate actual height in the "leaflet container container". Which, by the way, shouldn't use `height:100vh`, since it makes it overflow the actual page, but let's leave it to another iteration.

Reply:
Alright, we can track this in another issue if we run into this problem. It is being tracked in #22